logic.py

```diff
@@ -0,0 +1,12 @@
+## conditons when we have list of possible values 
```

Comment:
GPT summary of [None](https://github.com/SampatSuthar/ml-demo/blob/cccc5e0aa50c29cda8434cc1a92463d295c7584b/logic.py#None) - [ff48b2](https://github.com/SampatSuthar/ml-demo/blob/0c97036f0373bc6c942bcdc96667a24a6851e90d/logic.py#ff48b223391a0063edf03dca03d22da628bb8158):
* Added code to check the petrol price for a given state
* If the state is one of Rajasthan, Gujrat, or Karnataka, the petrol price will be 108.18
* If the state is Maharashtra, the petrol price will be 111.30
* If the state is neither of the above, the petrol price will be 110.00